example/frontend-testing/processor.js

```diff
@@ -0,0 +1,34 @@
+class AudioProcessor extends AudioWorkletProcessor {
+  constructor() {
+    super();
+    this.sampleRate = sampleRate; // Default: 16000 Hz
+    this.chunkSize = 8000; // Default: 0.1 seconds at 16 kHz / 8000 samples: 0,5 seconds
+    this.buffer = [];
+  }
+
+  process(inputs) {
+    const input = inputs[0];
+    if (input && input[0]) {
```

Comment:
this is kind of a weird check, no?
why not just do the input[0] and do the assigning after?

Reply:
no

Reply:
reasonable